src/js/background.js

```diff
@@ -32,6 +32,8 @@ chrome.webRequest.onBeforeRequest.addListener(
       '*://*.usconservativetoday.com/*',
       '*://*.worldnewsdailyreport.com/*',
       '*://*.worldpoliticus.com/*',
+      '*://*.nationalreport.net/*',
+      '*://*.newsexaminer.net/*'
```

Comment:
Argh, I see that you fell victim of a shortcut I took during the hackathon. There's actually two block lists, one in [`background.js`](https://github.com/jacquerie/stop-the-bullshit/blob/4b3f7a5569f57647893bff63b1a93b55ca00aa5f/src/js/background.js#L10-L35) and one in [`facebook.js`](https://github.com/jacquerie/stop-the-bullshit/blob/4b3f7a5569f57647893bff63b1a93b55ca00aa5f/src/js/facebook.js#L2-L26).

They actually (should) mirror their content, so this part should be refactored to use a common list.